archive_verify/workers.py

```diff
@@ -104,19 +107,34 @@ def verify_archive(archive, archive_path, description, config):
     dest_root = config["verify_root_dir"]
     dsmc_log_dir = config["dsmc_log_dir"]
     whitelist = config["whitelisted_warnings"]
+
+    now_str = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M')
+    log.setLevel(logging.DEBUG)
+    fh = logging.FileHandler("{}/{}-{}.log".format(dsmc_log_dir, description, now_str))
```

Comment:
nitpicking of the day: `os.path.join(dsmc_log_dir, "{}-{}.log".format(description, now_str))`

Reply:
I don't feel that it is a nitpick, you are entirely correct in pointing that out.
Well this is embarrassing :sweat: